src/repl/session.rs

```diff
@@ -536,6 +343,47 @@ impl Session {
         }
     }
 
+    pub fn get_boot_contracts_asts(&self) -> BTreeMap<QualifiedContractIdentifier, ContractAST> {
+        let mut asts = BTreeMap::new();
+        let (pox_ast, _, _) = self.interpreter.build_ast(
+            QualifiedContractIdentifier::transient(),
+            POX_CONTRACT.to_string(),
+            2,
+        );
+        let (bns_ast, _, _) = self.interpreter.build_ast(
+            QualifiedContractIdentifier::transient(),
+            BNS_CONTRACT.to_string(),
+            2,
+        );
+
+        let bns_contract = ContractName::try_from("bns").unwrap();
+        let pox_contract = ContractName::try_from("pox").unwrap();
+
+        let boot_testnet =
+            PrincipalData::parse_standard_principal("ST000000000000000000002AMW42H").unwrap();
+
+        let boot_mainnet =
+            PrincipalData::parse_standard_principal("SP000000000000000000002Q6VF78").unwrap();
```

Comment:
Would it make sense to move these into lazy statics?

Reply:
Addressed with [latest](https://github.com/hirosystems/clarity-repl/pull/159/commits/257b12af8673c56d8112232e2489b564beaf5755)